cardano-sl.cabal

```diff
@@ -43,10 +31,12 @@ library
                         Pos.Block.BListener
                         Pos.Block.Core
                         Pos.Block.Logic
+                        Pos.Block.Logic.Util
                         Pos.Block.Network
                         Pos.Block.Pure
                         Pos.Block.RetrievalQueue
                         Pos.Block.Slog
+                        Pos.Block.Slog.Types
```

Comment:
To be honest, I wouldn't expose `Pos.Block.Logic.Util` and `Pos.Block.Slog.Types`, they are available from `Pos.Block.Logic` and `Pos.Block.Slog` respectively.